app/controllers/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UsersController < ApplicationContoller
+  def index; end
+
+  def shoe; end
```

Comment:
- [ ] You have added `shoe` method here instead of using `show` method according to [project guidelines](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project.md).